gtest-parallel

```diff
@@ -127,17 +128,26 @@ class FilterFormat:
 
   tests = {}
   outputs = {}
-  started = []
-  finished = []
   failures = []
+  finished = []
+
+
+  def write_logs_to_file(self, bundle_name, tests_ids):
+    with open(os.path.join(self.logs_dir, bundle_name), 'w') as logs_bundle:
+      for test_id in tests_ids:
+        log_file = self.outputs[test_id]
+        test_name = os.path.basename(log_file)
+        logs_bundle.write('TEST: ' + test_name + '\n\n')
+        with open(self.outputs[test_id]) as test_log:
+          logs_bundle.write(test_log.read())
+        logs_bundle.write('\n\n')
 
   def print_tests(self, message, test_ids):
```

Comment:
I don't think we need the message. Can't we just make it simpler and just output the test output? The filename showcases what the content represents (and since the tests will only show RUN and not OK for interrupted tests, I think everyone will understand).